cni/network/network.go

```diff
@@ -435,6 +435,7 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 		// previously we had a default interface info to select which interface info was the one to be returned from cni add
 		cniResult := &cniTypesCurr.Result{}
 		for key := range ipamAddResult.interfaceInfo {
+			logger.Info("Exiting add, interface info retrieved", zap.Any("ifInfo", ipamAddResult.interfaceInfo[key]))
```

Comment:
Can we move this into its own dedicated loop-- currently if we find the infra nic we don't print out the rest of the interfaces